### ISSUE TITLE
`optic-ci compare` no longer works

### ISSUE THREAD

**Issue by cmars** [snyk/sweater-comb#114]
Regression possibly introduced by #112.

`optic-ci compare` no longer works. To reproduce:

```
$ yarn run compare --to ./end-end-tests/api-standards/resources/thing/2021-11-10/000-baseline.yaml 

Loading specifications for comparison:
Current specification: done
Next specification: done


  FAIL   Entire Resource
  x api lifeycle:  must lifeycle rules have to be followed
    Cannot read property 'date' of undefined
    Read more in our API guide (​https://github.com/snyk/sweater-comb/blob/main/docs/version.md​)
    at (/Users/cmars/Projects/sweater-comb/end-end-tests/api-standards/resources/thing/2021-11-10/000-baseline.yaml:1:0)
  x api lifeycle:  must follow sunset rules
    Cannot read property 'stability' of undefined
    at (/Users/cmars/Projects/sweater-comb/end-end-tests/api-standards/resources/thing/2021-11-10/000-baseline.yaml:1:0)


  PASS   Resource Document


  PASS   This Specification


  PASS  DELETE /orgs/{org_id}/thing/{thing_id}


  PASS  GET /orgs/{org_id}/thing


  PASS  GET /orgs/{org_id}/thing/{thing_id}


  PASS  PATCH /orgs/{org_id}/thing/{thing_id}


  PASS  POST /orgs/{org_id}/thing





1728 checks passed
2 checks failed

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by acunniffe** [comment]
Landing something for this shortly.

**Comment by acunniffe** [comment]
Cleanup: Should be fixed in #117
